TA_dataset/bin/dataset_query.py

```diff
@@ -144,7 +144,19 @@ def stream_events(self, inputs, ew):
             proxy = get_proxy(session_key, logger)
             acct_dict = get_acct_info(self, logger, ds_account)
             for ds_acct in acct_dict.keys():
+                if acct_dict.get(ds_acct).get("tenant") is not None:
```

Comment:
would it be possible to extract this code duplication to separate method?

Reply:
this is called with in the dataset_query file only, even if we do separate method this method has only one calling method in this file

Reply:
There are 3 similar blocks
- https://github.com/scalyr/dataset-addon-for-splunk/pull/127/files#diff-d18f827fd8bcb2d6d19de613abcf89f511869f92e201edc5521bb16705880e7cR123
- https://github.com/scalyr/dataset-addon-for-splunk/pull/127/files#diff-82c2c47a7187c5d5ae31339c5b9a97eb6760d164afac8c973e29d5ed4c95f200R116
- https://github.com/scalyr/dataset-addon-for-splunk/pull/127/files#diff-61e399a15e16c531564f016785921ec4e678e17ec74380722f9abf2788de528fR150

Reply:
Done